AppMusicA/AppMusicA/ViewController.swift

```diff
@@ -90,5 +90,55 @@ final class ViewController: UIViewController {
         let audioDuration = player.duration
         let newPlayTime = audioDuration * Double(sender.value)
         player.currentTime = newPlayTime
+        setupNowPlayingInfo()
     }
-}
\ No newline at end of file
+    private func setupRemoteCommandControls() {
+        let commandCenter = MPRemoteCommandCenter.shared()
+        commandCenter.pauseCommand.isEnabled = true
+        commandCenter.pauseCommand.addTarget { [unowned self] _ in
+            player.play()
+            return .success
+        }
+        commandCenter.playCommand.isEnabled = true
+        commandCenter.playCommand.addTarget { [unowned self] _ in
+            player.pause()
+            return .success
+        }
+        commandCenter.nextTrackCommand.isEnabled = true
+        commandCenter.nextTrackCommand.addTarget { [unowned self] _ in
+            if possition < audioList.count {
+                possition += 1
+                loadOtherAudio()
+            }
+            return .success
+        }
+        commandCenter.previousTrackCommand.isEnabled = true
+        commandCenter.previousTrackCommand.addTarget { [unowned self] _ in
+            if possition > 0 {
+                possition -= 1
+                loadOtherAudio()
+            }
+            return .success
+        }
+    }
+    func setupNowPlayingInfo() {
+        let currentAudio = audioList[possition]
+        let SIZE = CGSize(width: 20, height: 20)
+        if let artwork = UIImage(named: audioList[possition].name) {
+            let mediaArtwork = MPMediaItemArtwork(boundsSize: SIZE) { _ in
+                return artwork
+            }
+            var nowPlayingInfo = MPNowPlayingInfoCenter.default().nowPlayingInfo
+            nowPlayingInfo = [
+                MPMediaItemPropertyTitle: currentAudio.title,
+                MPMediaItemPropertyArtist: currentAudio.performer,
+                MPMediaItemPropertyPlaybackDuration: player.duration, // Thời lượng bản nhạc (giây)
+                MPNowPlayingInfoPropertyElapsedPlaybackTime: player.currentTime, // Thời gian đã phát (giây)
+                MPNowPlayingInfoPropertyPlaybackRate: player.rate, // Tốc độ phát
+                MPMediaItemPropertyArtwork: mediaArtwork // Hình ảnh album
```

Comment:
chat GPT đúng không, toàn cmt =))

Reply:
công cuộc mò xem cấu hình MPNowPlaying đủ chưa mà nó không hiện đấy a xD